docs/int/quickstart/group/quickstart-group-leader-creator.md

```diff
@@ -68,7 +85,24 @@ You will prepare the configuration file for the distributed key generation cerem
 4. Follow the flow and accept the advisories.
 5. Configure the Cluster 
     - Input the `Cluster Name` & `Cluster Size` (i.e. number of operators in the cluster). The threshold for the cluster to operate sucessfully will update automatically.
-    - ⚠️ Leave the `Non-Operator` toggle off if you are a **Leader** and will participate in the cluster. Turn the toggle on if you are a **Creator** and won't participate.
+    
+    <Tabs groupId="leader-creator">
+      <TabItem value="leader" label="Leader" default>
+    <ul><li>
+    
+    ⚠️ Leave the `Non-Operator` toggle OFF.
+    
+    </li></ul>
+      </TabItem>
+      <TabItem value="creator" label="Creator">
+    <ul><li>
+    
+    ⚠️ Turn the `Non-Operator` toggle ON.
```

Comment:
```suggestion
    ⚠️ Turn the `Non-Operator` toggle <b>ON</b>.
```

Reply:
i do think we should have a rethink of that button, it's currently too much of a double negative imo